fe/fe-core/src/main/java/com/starrocks/storagevolume/StorageVolume.java

```diff
@@ -229,15 +237,29 @@ public static Map<String, String> getParamsFromFileStoreInfo(FileStoreInfo fsInf
             case HDFS:
                 // TODO
             case AZBLOB:
-                // TODO
+                AzBlobFileStoreInfo azBlobFileStoreInfo = fsInfo.getAzblobFsInfo();
+                params.put(CloudConfigurationConstants.AZURE_BLOB_ENDPOINT, azBlobFileStoreInfo.getEndpoint());
+                AzBlobCredentialInfo azBlobcredentialInfo = azBlobFileStoreInfo.getCredential();
+                String sharedKey = azBlobcredentialInfo.getSharedKey();
+                if (!Strings.isNullOrEmpty(sharedKey)) {
+                    params.put(CloudConfigurationConstants.AZURE_BLOB_SHARED_KEY, sharedKey);
+                }
+                String sasToken = azBlobcredentialInfo.getSasToken();
+                if (!Strings.isNullOrEmpty(sasToken)) {
+                    params.put(CloudConfigurationConstants.AZURE_BLOB_SAS_TOKEN, sasToken);
+                }
+                return params;
             default:
                 return params;
         }
     }
 
-    private void setEmptyAuthenticationIfNeeded(Map<String, String> params) {
+    private void preprocessAuthenticationIfNeeded(Map<String, String> params) {
         if (svt == StorageVolumeType.HDFS) {
             params.computeIfAbsent(HDFS_AUTHENTICATION, key -> HDFSCloudCredential.EMPTY);
+        } else if (svt == StorageVolumeType.AZBLOB) {
+            String container = locations.get(0).split("/")[0];
```

Comment:
Do we already check `locations.get(0)` must contains `\`?

Reply:
> Do we already check `locations.get(0)` must contains `\`?

For `abc`, it will return `abc`.

Reply:
I have added tests in `StorageVolumeTest`.